index.php

```diff
@@ -78,12 +80,21 @@
 
 function listLayers(string $version, string $region): array
 {
-    $client = new Client();
-    $url = 'https://raw.githubusercontent.com/brefphp/bref/' . $version . '/layers.json';
+    $cache = new FilesystemAdapter();
```

Comment:
Where are the files stored?

It should be in `/tmp`, everything else is readonly on AWS Lambda.

Reply:
It will use `sys_get_temp_dir()` to determine the proper location. Will that work on AWS Lambda or should I make it explicit?

Reply:
You know what let's try it out ^^